src/lib.rs

```diff
@@ -159,6 +159,13 @@ pub struct Package {
     pub name: String,
     /// Version given in the `Cargo.toml`
     pub version: String,
+    /// The [SPDX license expression][spdx-expressions] for this package.
+    ///
+    /// [spdx-expressions]: https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60
+    pub license: Option<String>,
+    /// If the package is using a nonstandard license, this key may be specified instead of
+    /// `license`, and must point to a file relative to the manifest.
+    pub license_file: Option<String>,
```

Comment:
This should probably be a `PathBuf` then

Reply:
I didn't make it a PathBuf because you can technically store any string in this field, not necessarily a path. I could make it a PathBuf, but I suppose there's a risk of it not deserializing correctly.

Reply:
We already have a bunch of things that can go awry. Let's assume ppl use this field somewhat sensibly